moc_openshift.py

```diff
@@ -32,10 +33,28 @@ def create_rolebindings(self, project_name, user_name, role):
     def update_rolebindings(self, project_name, role, rolebindings_json):
         return
 
+    @abc.abstractmethod
+    def delete_moc_quota(self, project_name):
```

Comment:
I am a little confused by the naming of these methods, because we have variously `<verb>_quota`, `<verb>_shift_quota`, and `<verb>_moc_quota`.

I think that `<verbo>_quota` operates on a single resourcequota object, while `<verb>_moc_quota` may operate on  multiple objects, but what about `<verb>_shift_quota`?

Reply:
shift quota should probably be resource_quota.  moc_quota is the list with the mangled quota names.

Reply:
I'll take a pass over it tomorrow and try to make the naming convention a bit more consistent.